README.md

```diff
@@ -57,6 +57,12 @@
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/zsh/code/ci/master/tree/)
 
 
+### E-Books
+
```

Comment:
I think this category should be `E-Book Readers` or `E-Books Manager` or maybe something better?